### PR TITLE
281 related subscriptions

### PR DESCRIPTION
**Pull request by DutchBen**
Issues: https://github.com/workfloworchestrator/orchestrator-ui/issues/281
- Includes no result box
- Adds an option to table columns that makes them unsortable

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WFONoResults/styles.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import type { EuiThemeComputed } from '@elastic/eui';
+
+export const getStyles = (theme: EuiThemeComputed) => {
+    const panelStyle = css({
+        display: 'flex',
+        padding: theme.base * 2,
+        flexDirection: 'row',
+        alignItems: 'center',
+        justifyContent: 'center',
+        gap: '8px',
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WFONoResults/styles.ts:11]: you could use the base value here too

File: packages/orchestrator-ui-components/src/components/WFOTable/WFOBasicTable/WFOTableHeaderCell.tsx [hunk 2]
```diff
@@ -6,24 +6,26 @@ import { WFOSortDirectionIcon } from './WFOSortDirectionIcon';
 export type WFOTableHeaderCellProps = {
     sortDirection?: SortOrder;
     onClick?: () => void;
+    sortable?: boolean;
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WFOTable/WFOBasicTable/WFOTableHeaderCell.tsx:9]: suggesting `isSortable`

File: packages/orchestrator-ui-components/src/components/WFOSubscription/WFORelatedSubscriptions.tsx [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+                            showLabel={true}
+                            label={t('hideTerminatedRelatedSubscriptions')}
+                            type="button"
+                            checked={hideTerminatedSubscriptions}
+                            onChange={toggleTerminatedSubscriptions}
+                        />
+                    </EuiFlexItem>
+                </EuiFlexGroup>
+                <EuiSpacer size="m" />
+                {(relatedSubscriptions && relatedSubscriptions.length > 0 && (
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WFOSubscription/WFORelatedSubscriptions.tsx:176]: Could be extracted to a const and/or you can use relatedSubscriptions?.length

**Review by ricardovdheijden — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WFONoResults/styles.ts (hunk 1 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WFONoResults/styles.ts:11]: Done

File: packages/orchestrator-ui-components/src/components/WFOTable/WFOBasicTable/WFOTableHeaderCell.tsx (hunk 2 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WFOTable/WFOBasicTable/WFOTableHeaderCell.tsx:9]: Good one. Done

**Pull request merged**